Sofa/framework/Helper/src/sofa/helper/init.cpp

```diff
@@ -50,6 +51,8 @@ SOFA_HELPER_API void cleanup()
 {
     if (!s_cleanedUp)
     {
+
+        system::PluginManager::getInstance().cleanup();
```

Comment:
I wonder if Sofa.Helper is the right place to call a plugin clean up...

Reply:
It was asked by @fredroy and it makes sens because the plugin manager belongs to Sofa.Helper

Reply:
Yes, but your goal is to clean up plugins, not the plugin manager. For example, you will clean up SofaPython3 inside the Sofa.Helper cleanup function. I am not saying it is not the right thing to do, I am just questioning ...

Reply:
My first though was to put it at the very end of the main function, both are justifiable. But I guess that conceptually, the cleanup order shouldn't matter, the simulation is over, we are just cleaning after us...

Reply:
Why not inside the destructor of PluginManager ? (if it is ever called 🥴)